include/psp2/power.h

```diff
@@ -81,14 +83,19 @@ SceBool scePowerIsBatteryCharging(void);
  */
 int scePowerGetBatteryLifePercent(void);
 
+/*GPU, WLAN/COM configuration setting */
+#define SCE_POWER_CONFIGURATION_MODE_A			0x00000080U /*GPU clock normal, can use WLAN/COM       */
+#define SCE_POWER_CONFIGURATION_MODE_B			0x00000800U /*GPU clock high,   can't use WLAN/COM     */
+#define SCE_POWER_CONFIGURATION_MODE_C			0x00010880U /*GPU clock high,   can use WLAN/COM,      but...   */
+
 /**
  * Set configuration mode ?
  *
  * @param mode - The mode to set
  *
  * @return ?
  */
-int scePowerSetConfigurationMode(int mode);
+int scePowerSetConfigurationMode(int conf);
```

Comment:
Change the documentation as well, and add `::ScePowerConfigurationMode`.

Reply:
where is the documention ?